Plugin: 明日方舟抽卡记录分析

**插件名称：**

明日方舟抽卡记录分析

**插件功能：**

明日方舟抽卡记录分析与导出

**PyPI 项目名：**

nonebot-plugin-arkrecord

**插件 import 包名：**

nonebot_plugin_arkrecord

**插件项目仓库/主页链接：**

https://github.com/zheuziihau/nonebot_plugin_arkrecord

**标签：**

[{"label":"明日方舟 ","color":"#c39191"},{"label":"游戏","color":"#c39191"},{"label":"抽卡","color":"#c39191"}]

# 📃 商店发布检查结果

> Plugin: 明日方舟抽卡记录分析

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot_plugin_arkrecord 的信息如下：
    name         : nonebot-plugin-arkrecord                                             
     version      : 1.7.0                                                                
     description  : Nonebot plugin for fetching and analyzing gacha records of arknights 
    
    dependencies
     - lxml *
     - matplotlib *
     - nonebot-adapter-onebot &gt;=2.0.0b1,&lt;3.0.0
     - nonebot2 &gt;=2.0.0b4,&lt;3.0.0
     - Pillow *
     - requests *
     - XlsxWriter *
插件 nonebot_plugin_arkrecord 加载出错：
02-26 04:00:18 [SUCCESS] nonebot | NoneBot is initializing...
02-26 04:00:18 [INFO] nonebot | Current Env: prod
02-26 04:00:19 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_arkrecord&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_arkrecord&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arkrecord/__init__.py&quot;, line 7, in &lt;module&gt;
    from .ark import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arkrecord/ark/__init__.py&quot;, line 2, in &lt;module&gt;
    from .ark_scrawl import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arkrecord/ark/ark_scrawl.py&quot;, line 3, in &lt;module&gt;
    from .ark_drawer import ArkImage
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arkrecord/ark/ark_drawer.py&quot;, line 14, in &lt;module&gt;
    from .ark_setting import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arkrecord/ark/ark_setting.py&quot;, line 14, in &lt;module&gt;
    raise RuntimeError(&quot;未配置数据库路径&quot;)
RuntimeError: 未配置数据库路径</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 明日方舟 -#c39191, 游戏-#c39191, 抽卡-#c39191。</li><li>✅ 项目 <a href="https://github.com/zheuziihau/nonebot_plugin_arkrecord">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot_plugin_arkrecord/">nonebot_plugin_arkrecord</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4642,5 +4642,28 @@
     "homepage": "https://github.com/Hoshinonyaruko/nonebot-plugin-sanae",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_arkrecord",
+    "project_link": "nonebot-plugin-arkrecord",
+    "name": "明日方舟抽卡记录分析",
+    "desc": "明日方舟抽卡记录分析与导出",
+    "author": "zheuziihau",
+    "homepage": "https://github.com/zheuziihau/nonebot_plugin_arkrecord",
+    "tags": [
+      {
+        "label": "明日方舟 ",
+        "color": "#c39191"
+      },
+      {
+        "label": "游戏",
+        "color": "#c39191"
+      },
+      {
+        "label": "抽卡",
+        "color": "#c39191"
+      }
+    ],
+    "is_official": false
   }
 ]

